Must find heat flow for a refrigerator around Sentani region and possible cost per bottle of water 
(or how long to break even)



Assuptions:
use $q=UA\Delta T$
Using Sanyo HF-S6L refridgerator for basis of calculations
average Jayapura weather= 29C
average refridge temp=1.6C

Stats of refirgierator: 
230V/ 60Hz
Demensions:W*L*H=545mm*693mm*1295mm=1.788ft*2.27ft*4.286ft

SA=42.59ft^2 or 3.9m^2
U of polystyrene

In [6]:
k = .03
#t= (a guess) 1 in and a half=.0381m

In [7]:
k / .0381

0.7874015748031495

In [5]:
(.7874)*(3.9)*(29-1.6)

84.14156399999999

84.14 watts of heat flow
Now we must find an assumed kwh cost of electricity which the average in indonesia is .11 USD (wiki 2015)
however we need to find optimal time fridge is used and amount of time (in hours) used
also we must find the amount of energy to cool beverage at optimal temperature
29C to 1.6C
$Q=mc\Delta T$
avg size of water bottle=16.9 oz or 479.1 grams
specific heat capacity=4.179
Change in temp=(29-1.6C)=27.4C

In [11]:
m = 479.1
c = 4.179
dT = 27.4

In [12]:
(m)*(c)*(dT)

54859.15386

54859.15 Joules to lower temperature of 1 bottle of water